otx/mpa/det/semisl/stage.py

```diff
@@ -37,36 +37,21 @@ def configure_data(self, cfg, data_cfg, training, **kwargs):
                 )
 
     def configure_task(self, cfg, training, **kwargs):
```

Comment:
Why do we inherit  `SemiSLDetectionStage` from IncrDetectionStage? I thought this PR should refactor inheritance and create base Stage. Other stages will be inherited from this Base one.

Reply:
Great point, I have a mistake here. Thank you for your comment